src/datafreetree.jl

```diff
@@ -0,0 +1,14 @@
+type DataFreeTree{T <: AbstractFloat, M <: Metric}
+    size::Tuple{Int64,Int64}
```

Comment:
Isn't this what is causing the test failures? On 32 bit `size(data)` will be a `Tupel{Int32, Int32}`. Shouldn't you just have Int here?